src/main/scala/au/com/agiledigital/toolform/command/generate/kubernetes/VolumeClaimWriter.scala

```diff
@@ -0,0 +1,106 @@
+package au.com.agiledigital.toolform.command.generate.kubernetes
+
+import au.com.agiledigital.toolform.model.{Resource}
+import collection.JavaConversions._
+import collection.JavaConverters._
+import com.typesafe.config._
+import cats.implicits._
+import scala.util.{Failure, Success, Try}
+
+object VolumeClaimWriter extends KubernetesWriter {
+
+  // write the various access modes
+  def writeAccessModes(resource: Resource, serviceName: String): Result[Unit] = {
+    val acceptableModes = List("ReadWriteOnce", "ReadOnlyMany")
+    val accessModes = resource.settings match {
+      case Some(aModes) => {
+        Try(aModes.getStringList("accessModes")) match {
+          case Success(list) => list
+          // no accessModes specified in project.conf
+          case Failure(error) => {
+            println(s"Warning: no access modes specified for resource: $serviceName. Default of ReadWriteOnce used")
```

Comment:
@timechanter @NoxHarmonium @dspasojevic Is there an existing mechanism for warning other than a simple println? Do we want to be warning in cases where defaults are assumed, or should the writers be silent.

Reply:
In my opinion warnings such as "access mode unsupported for resource" might be better as errors so that we enforce valid configs and warnings where defaults are assumed should be silent because of its expected behaviour.